app/viewers/embed/viewer/common_viewer.rb

```diff
@@ -112,6 +112,15 @@ def iframe_title
         I18n.t('title', default: 'Viewer', scope: i18n_path, title: purl_object.title)
       end
 
+      def message
+        if @purl_object.location_restriction
+          return { type: 'location-restricted',
+                   message: I18n.t('restrictions.restricted_access', location: @purl_object.restricted_location) }
+        end
+
+        false
```

Comment:
```suggestion
        { type: 'location-restricted',
                   message: I18n.t('restrictions.restricted_access', location: @purl_object.restricted_location) } if @purl_object.location_restriction
```

Reply:
I am pretty sure this is how I originally wrote this and rubocop corrected to this. I can double check.

Reply:
Yup. Rubocop complained. It is back to the way it was. ```Style/MultilineIfModifier: Favor a normal if-statement over a modifier clause in a multiline statement```